ios/Mattermost.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,24 +1,6 @@
 {
   "object": {
     "pins": [
-      {
-        "package": "OpenGraph",
-        "repositoryURL": "https://github.com/satoshi-takano/OpenGraph.git",
-        "state": {
-          "branch": null,
-          "revision": "382972f1963580eeabafd88ad012e66576b4d213",
-          "version": "1.4.1"
-        }
-      },
-      {
-        "package": "Sentry",
-        "repositoryURL": "https://github.com/getsentry/sentry-cocoa.git",
-        "state": {
-          "branch": "8.15.2",
```

Comment:
you should bump this one to match the one with the current sentry RN package which is 8.21.0